lib/system/ansi_c.nim

```diff
@@ -116,9 +116,15 @@ type
           incompleteStruct.} = object
   CFilePtr* = ptr CFile ## The type representing a file handle.
 
+# duplicated between io and ansi_c
+const stderrName* = when defined(osx): "__stderrp" else: "stderr"
+const stdoutName* = when defined(osx): "__stdoutp" else: "stdout"
+const stdinName* = when defined(osx): "__stdinp" else: "stdin"
```

Comment:
Why are these exported? In ANSI C it's called ``stdout`` btw, not ``__stdoutp``.

Reply:
* fixed (was leftover from removing imports_common.nim)
> In ANSI C it's called stdout btw, not __stdoutp

on OSX, stdout is a macro:
```
#define stdout  __stdoutp
```
the real symbol is `__stdoutp`; so `dlopen` will work with `__stdoutp` but not `stdout`.
You can test this with `tests/vm/tevalffi.nim` (requires `-d:nimHasLibFFI`)